### ISSUE TITLE
Set max PA Level

### ISSUE THREAD

**Issue by jaimeambrosio** [dparson55/NRFLite#12]
How to assign the maximum power level to achieve a greater distance ?. Similar to radio.setPALevel (RF24_PA_MAX)

**Comment by dparson55** [comment]
NRFLite sets the RF output power to the maximum of 0dBm and I did not expose a feature to adjust it.  I did this to keep the library as simple as possible since the power consumption among the various output power settings is insignificant in most real world situations.  The power is configured as part of the RF_SETUP register within `initRadio` in [NRFLite.cpp](https://github.com/dparson55/NRFLite/blob/master/src/NRFLite.cpp).

At the lowest RF output power setting (-18dBm) the radio consumes around 7mA while at the highest output power (0dBm) it uses 12mA.  This power consumption is brief however, it only occurs for microseconds when a data packet is actually transmitted.  When no data is being sent the radio is placed into Standby-I mode which only consumes 26uA, no matter the selected RF output power.

For grid powered projects the 5mA difference is insignificant.  For battery powered projects that send data periodically, the 5mA difference isn't going to affect the size of the required battery and you will need to do a lot of power management either way (see [Sensor_TX_ATtiny85](https://github.com/dparson55/NRFLite/tree/master/examples/Sensor_TX_ATtiny85_2Pin) for an example).  For battery powered projects that constantly stream data, the 5mA difference will likely be overshadowed by other components like the microcontroller, but this is the one situation where a lower output power could make sense.  For such a case though, NRFLite is not a good fit because it trades efficiency for ease-of-use and for a battery powered, constantly streaming data project, you'd definitely want efficiency rather than the ease-of-use that NRFLite provides.

So anyway, that is why I didn't expose the setting and just made it transmit at its maximum power.
